js/initcontext.go

```diff
@@ -164,16 +164,35 @@ func (i *InitContext) compileImport(src, filename string) (*goja.Program, error)
 	return pgm, err
 }
 
-func (i *InitContext) Open(name string, args ...string) (goja.Value, error) {
-	filename := loader.Resolve(i.pwd, name)
+// Open implements open() in the init context and will read and return the contents of a file
+func (i *InitContext) Open(filename string, args ...string) (goja.Value, error) {
+	// Here IsAbs should be enough but unfortunately it doesn't handle absolute paths starting from
+	// the current drive on windows like `\users\noname\...`. Also it makes it more easy to test and
+	// will probably be need for archive execution under windows if always consider '/...' as an
+	// absolute path.
+	if filename[0] != '/' && filename[0] != '\\' && !filepath.IsAbs(filename) {
```

Comment:
Ah, I just realized that calling `open()` or `open('')` or `open(undefined)` will result in a panic... That's how it currently is too... :man_facepalming: Please, return a normal error in those cases and add a test for them